lib/services/init_service.dart

```diff
@@ -115,12 +119,15 @@ class InitService {
 
     // 3. For each company fetch direct chats
     final directFutures = companies.map((c) async {
-      remoteResult = await _apiService.get(
+      remoteResult = await _apiService
```

Comment:
This should not be changed here, it seems by auto format from your IDE. Check again in another places